lua/lean/_util.lua

```diff
@@ -107,6 +107,9 @@ end
 
 M.a_request = a.wrap(M.request, 4)
 
+-- FIXME: tick locking is disabled for now
```

Comment:
I think a better alternative to this is using a variant of `lsp.buf_request` that has a built-in timer after which it expires and calls the handler with an error (not sure if this already exists).

Reply:
That's what I tried first, but I couldn't get it working (without spending too much time on it).  For now, I think it's less annoying if the infoview is sometimes corrupted than if it stops updating completely.

Reply:
Right, but doesn't 6d8314c fix it without this?

Reply:
I've run into similar problems (infoview not updating) with Lean 4 as well, but they're harder to reproduce.

Reply:
Interesting. And you're sure the the Lean 4 server is guaranteed to respond to every LSP request that we're making here? And when it did halt in Lean 4, did you get an error?

I'm just concerned that doing this -- in addition to the bugs it introduces -- serves to hide issues that we need to solve, here or possibly even in the server itself.